.env.sample

```diff
@@ -0,0 +1,4 @@
+DB_HOST=DB_HOST
+DB_NAME=DB_NAME
+DB_USER=DB_USER
+DB_PASSWORD=DB_PASSWORD
```

Comment:
The variables should be named named with `POSTGRES_` prefix as is required in the official postgres image https://hub.docker.com/_/postgres